### PR TITLE
Add content-copy icon

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

**Review by 3imed-jaberi — COMMENTED** [review]
Ready to 👀

**Review by Umerbhat — CHANGES_REQUESTED** [review]
Update package version

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "paack-ui-assets",
-  "version": "0.4.2",
+  "version": "0.4.3",
```
Comment (3imed-jaberi) [inline package.json:3]: bumping based on the prev PR.

> I see that should bum the minor number, not the patch!

**Review by Mbes9 — APPROVED** [review]

**Review by Umerbhat — APPROVED** [review]

**Review by 3imed-jaberi — COMMENTED** [review]
Done ✅

**Pull request merged**